graphistry/feature_utils.py

```diff
@@ -111,7 +140,16 @@ def assert_imported():
         )
         raise import_min_exn
 
-
+def assert_cuml_imported():
```

Comment:
`assert_cuml_cucat` ?

The reason Im being a stickler is I think we have code clones across files, so we need to avoid divergence bugs if folks start editing them and get confused bc they are called the same